test/services/helpdesk/index.test.js

```diff
@@ -89,5 +94,29 @@ describe('helpdesk service', function() {
 			}
 		);
 	});
+
+	it('DELETE /helpdesk with user from same school', () => {
+
+		helpdeskService.remove('5836bb5664582c35df3bc214', { payload: {userId: '0000d213816abba584714c0a'}})
```

Comment:
leider beweist du durch beide Tests gar nichts.
Wenn du dir den Hook Mal genau anguckst, dann wirst du feststellen, dass er in Zeile 206/207 springen wird von `src/hooks/index.js`. Du fügst also einem Delete Request eine SchoolId hinzu, welche keine weiteren Auswirkungen auf den Verlauf hat.
Die if Condition guckt ob deine im Param angegebene schoolId mit dem des Users übereinstimmt, aber da keine gegebene ist, ist der Test auch nicht aussagekräftig.